search-resources-dynamodb-trigger/src/test/java/no/unit/nva/publication/TestDataGenerator.java

```diff
@@ -55,8 +58,11 @@ public InputStream createResourceEvent(String eventType,
                                            PublicationStatus newPublicationStatus)
         throws JsonProcessingException, MalformedURLException, InvalidIssnException {
         initTemplate();
-
-        return generateModifyEvent(oldPublicationStatus, newPublicationStatus, eventType);
+        if (REMOVE.equals(eventType)) {
+            return generateRemoveEvent(oldPublicationStatus);
```

Comment:
The action is decided on the content. If it contains a new image it is an update (insertion is treated as an update). If it contains only an old image it is a deletion.